src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+// Write action creators to add and remove books
+export const addBook = (book) => ({
+  type: 'BOOK_ADDED',
```

Comment:
- [ ] kindly add action types and then use them in your action creators, something like this: `const BOOK_ADDED = bookstore/books/ADD_BOOK`. Let's do the same to the `BOOK_REMOVED`